package-testing/pre/000_install_package.rb

```diff
@@ -2,13 +2,24 @@
   workstation = find_at_most_one('workstation')
 
   step 'Install pdk package' do
+    if ENV['LOCAL_PKG']
+      pkg = File.basename(ENV['LOCAL_PKG'])
+      scp_to(workstation, ENV['LOCAL_PKG'], pkg)
```

Comment:
It seems like we get a reasonable error message already:

    Install pdk package on workstation host
    
      * Install pdk package
        localhost $ scp john_cena.msi windows2012r2-64-1:john_cena.msi {:ignore => }
        IOError: No such file or directory - john_cena.msi
        /home/centos/code/pdk/package-testing/pre/000_install_package.rb:7
        /home/centos/code/pdk/package-testing/pre/000_install_package.rb:4

I will add a File.exist? check to the Rakefile though, as it's nicer to abort immediately before beaker spends time on setting up VMs